kamelet-support/src/test/resources/io/kaoto/backend/api/service/step/parser/kamelet/eip.kamelet.yaml

```diff
@@ -65,20 +65,20 @@ spec:
               exception-type: java.lang.IllegalArgumentException
               message: test
           - routing-slip:
-              uri-delimiter: '|'
               simple: ${body}
+              uri-delimiter: '|'
```

Comment:
What is the link to the Pull request for the modification in this file?

Reply:
I am not sure I understand. But the order of the properties changed because I standardized the way expressions are treated across different EIPs.

Reply:
relates to https://github.com/KaotoIO/kaoto-backend/pull/510#discussion_r1126491779